src/app/scripts/cac/control/cac-control-trip-options.js

```diff
@@ -270,6 +285,108 @@ CAC.Control.TripOptions = (function ($, Handlebars, moment, Modal, UserPreferenc
+        ].join('');
+
+        var bikeShare = options.defaultMenuText.bikeShare;
+        if (!UserPreferences.isDefault('bikeShare')) {
+            var useBikeShare = UserPreferences.getPreference('bikeShare');
+            if (useBikeShare) {
+                bikeShare = 'Use Indego bike sharing';
+            } else {
+                bikeShare = 'Use my own bike';
+            }
+        }
+
+        var bikeTriangle = options.defaultMenuText.bikeTriangle;
+        if (!UserPreferences.isDefault('bikeTriangle')) {
+            var bikePreference = UserPreferences.getPreference('bikeTriangle');
+            bikeTriangle = [
+                bikePreference.charAt(0).toUpperCase(),
+                bikePreference.slice(1),
+                ' ride'
+            ].join('');
+        }
+
+        var timing = getTimingText();
+
+        var template = Handlebars.compile(source);
+        var html = template({
+            bikeShare: bikeShare,
+            timing: timing,
+            bikeTriangle: bikeTriangle
+        });
+
+        return html;
+    }
+
+        /**
+     * Helper to generate options list for top-level menu to display when bike mode is off.
+     * Will display current selection instead of default text if user has explicitly set an option.
+     *
+     * @returns {String} HTML snippet with list items to replace content inside
+                         walk modal's unordered list
+     */
+    function walkModalOptions() {
+
+        var source = [
+            '<li class="modal-list-timing">{{timing}}</li>',
+            '<li class="modal-list-accessibility">{{accessibility}}</li>'
+        ].join('');
+
+        var accessibility = options.defaultMenuText.accessibility;
+        if (!UserPreferences.isDefault('wheelchair')) {
+            var wheelchair = UserPreferences.getPreference('wheelchair');
+            if (wheelchair) {
+                accessibility = 'I have a wheelchair';
+            } else {
+                accessibility = 'I am walking';
+            }
+        }
+
+        var timing = getTimingText();
+
+        var template = Handlebars.compile(source);
+        var html = template({
+            accessibility: accessibility,
+            timing: timing,
+        });
+
+        return html;
+    }
+
+    /**
+     * Helper to get the modal dialog text for arrival/departure time,
+     * shared by bike and walk modals.
+     *
+     * @returns {string} Text snippet to display
+     */
+    function getTimingText() {
+        var timing = options.defaultMenuText.timing;
+        if (!UserPreferences.isDefault('dateTime')) {
+            var dateTime = moment.unix(UserPreferences.getPreference('dateTime'));
+            var arriveBy = UserPreferences.getPreference('arriveBy');
+            timing = arriveBy ? 'Arrive ' : 'Depart ';
+            // format as relative time
+            // TODO: something more exact
+            timing += dateTime.fromNow();
```

Comment:
The prototype shows this as an absolute time ("Depart 12/14 11:00a"), i.e. `dateTime.format('M/YY h:mma')`